internal/api/permissions.go

```diff
@@ -129,6 +130,17 @@ type checkResult struct {
 	Error   error
 }
 
+type bulkCheckActionsRequest []checkAction
+
+type checkActionResponse struct {
+	ResourceID    string `json:"resource_id"`
+	Action        string `json:"action"`
+	HasPermission bool   `json:"has_permission"`
```

Comment:
I think I have a _minor_ preference for a boolean result to be named something like `Allowed` given that we have some consumer-facing stuff that refers to access check results as "allowed" or "denied" elsewhere (like https://github.com/metal-toolbox/iam-runtime/blob/main/proto/authorization/authorization.proto#L40-L44) but otherwise 👍

Reply:
I agree, we should keep similar terminology.